app/components/contact-page-components/ContactDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
```

Comment:
I apologize if I didn't mention this earlier, but be sure to remove any unnecessary imports such as `import React from "react"`. :)  
_In the latest version of React, starting from version 17, you no longer need to explicitly import the React module in files that contain React components. This change was introduced as part of the "React 17 - No Breaking Changes" release._

Reply:
Whoops. One day I'll get them all